test/integration/azureService.js

```diff
@@ -3,12 +3,13 @@ const chai = require('chai');
 const azureService = require('../../app/lib/azureService');
 
 const expect = chai.expect;
-const name = 'test-file.json';
+// writing to a shared area, timestamp file to avoid collisions
+const name = `${new Date().getTime()}test.json`;
+console.log(name);
```

Comment:
If this is intentional can some additional info be added to the console output so it is easier to identify in the logs please?